packages/twenty-server/src/engine/api/graphql/graphql-query-runner/helpers/process-aggregate.helper.ts

```diff
@@ -19,17 +15,18 @@ export class ProcessAggregateHelper {
     for (const [aggregatedFieldName, aggregatedField] of Object.entries(
       selectedAggregatedFields,
     )) {
-      const fieldMetadata = fieldMetadataMapByName[aggregatedField.fromField];
-
-      if (!fieldMetadata) {
+      if (
+        !aggregatedField?.fromColumnName ||
+        !aggregatedField?.aggregationOperation
+      ) {
         continue;
       }
 
-      const fieldName = fieldMetadata.name;
+      const columnName = aggregatedField.fromColumnName;
```

Comment:
i would not talk about columns at this level (api). I think we should talk about subFields

Reply:
@charlesBochet we are building a SQL request here so I think it makes sense to talk about columns?

Reply:
I don't agree! The ORM is building SQL, but here we are above, we are building things to put in the ORM (~entity schemas that are close to metadata), no?